modules/nw-operator-cr.adoc

```diff
@@ -311,6 +311,10 @@ If you set this field to `true`, you do not receive the performance benefits of
 |`ipForwarding`
 |`object`
 |You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `ipForwarding` specification in the `Network` resource. Specify `Restricted` to only allow IP forwarding for Kubernetes related traffic. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. For updates to {product-title} 4.14 or later, the default is `Global`.
+[NOTE]
```

Comment:
Notes in tables can be a bit overwhelming in tables. Is a note needed here when the sentence could be tagged to the line for or near the first `Restricted` entry?

Reply:
Agree about the callouts in tables. Unfortunately, this is information that should not be ignored so it needs some kind of callout or IP forwarding on all new installations will drop.